setup.cfg

```diff
@@ -68,7 +68,7 @@ universal = 1
 
 [compile_catalog]
 directory = invenio_admin/translations/
-use-fuzzy = True
+use_fuzzy = True
```

Comment:
you could change every `-` to `_` on the left side of the `=`. in the other packages like rdm-records we only left `source-dir` and `build-dir` with the `-`, but i think the reason for it, is that we didn't find a problem there yet.

so `mapping-file` to `mapping_file` etc